my-app/src/main/java/com/JavaWebServer/app/Request.java

```diff
@@ -10,11 +10,30 @@ public Request () {
 
   }
 
+  public boolean isParams() {
```

Comment:
Name is somewhat misleading. Is the Request params? Or does the Request have params?